### DISCUSSION TITLE
Cypress not able to connect to Edge while executing in command line mode

### DISCUSSION THREAD

**Discussion by parulbhagat-soprabanking** [cypress-io/cypress#28069] [Questions and Help]
Cypress not able to connect to Edge while executing in command line mode. It gives the below popup asking for admin credentials and also the error message on cypress command line saying unable to connect to cypress:
Is there any configuration from where I can disable this popup?

This has started to happen after teh latest windows / edge update.

![image](https://github.com/cypress-io/cypress/assets/131943336/df8640c7-0fcd-443d-9412-0f2cb56f30e4)
![image](https://github.com/cypress-io/cypress/assets/131943336/a464f85a-7c37-4849-aad2-c2f887ddac6a)

**Comment by sergavt** [comment]
Has anyone figured this out yet?   Once  I found the workaround this after I downgraded cypress to v 12.4, however later the Edge was updated again and this issues came back again 
Details:
Cypress package version: 12.4.0
Cypress binary version: 12.4.0
Electron version: 21.0.0
Bundled Node version:
16.16.0
